saleor/core/middleware.py

```diff
@@ -35,6 +36,10 @@ def process_request(self, request):
 class CountryMiddleware(object):
 
     def process_request(self, request):
+        cookie = request.COOKIES.get(COOKIE_COUNTRY)
```

Comment:
Language is switched using session, should we use session instead of cookies here?